### ISSUE TITLE
Edit in Pop-up-windows

### ISSUE THREAD

**Issue by luenissla** [fisharebest/webtrees#4821]
Hello Greg, 
I know that you develop webtrees with special attention to mobile devices. This is also evident in the handling of version 2. 

Unfortunately, this means that the input masks are no longer offered in pop-up windows, but as a full screen. This was always very nice in version 1 because you could move the window if you wanted to transcribe a document on your own PC or online. These pop-up windows were very stable and did not collapse when you switched between windows. That was excellent!

Would it be possible to get these pop-up input windows again, at least via an option? It would be very helpful. 

There are now more and more ways to view certificates online and you need two browser windows.
Regards
Hans-Joachim

**Comment by Norwegian-Sardines** [comment]
What I do when working on my computer is instead of left clicking on the link I right click and open the page in a new tab.

This allows for me to get back to the prior screen with ease if it is a search list.  I also can open multiple copy’s of the site in the same browser session or in a new browser.  

If you are using a laptop you can get a small second monitor and have two browsers running each on their own screen.

I’m lucky I have 2 large screens on my desk, but in Windows10 and 11 if you have one large screen you can tell Win to treat it like two screens and run multiple browsers that way!

**Comment by luenissla** [comment]
Of course, it is obvious to use the context menu. However, it means that more clicks are needed and the new window is built up with full content (header, menus, footer). However, I don't think of myself in terms of operation; whereas, unfortunately, I don't have two screens available. I'm also thinking of the other users, who are usually older and not so familiar with the technology. Hence my question about the possibility of switching (sometimes as a full window (as in version 2) or only as a pop-up (as in version 1)). 

My dream would be to be able to integrate the functions of the DES of the Association for Computer Genealogy into Webtrees. [http://des.genealogy.net/ ](http://des.genealogy.net/ )
When mapping civil status documents and church records, data always accumulates that Webtrees allows to be entered in different input masks. It would certainly be possible to enter some data together and the programme would then divide them up accordingly.  That is certainly a vision for the future. But one can dream ;-)
